### ISSUE TITLE
resolve source artifacts optionally

### ISSUE THREAD

**Issue by andreak** [random-maven/scalor-maven-plugin#5]
I have this declared:
```
<plugin>
	<groupId>com.carrotgarden.maven</groupId>
	<artifactId>scalor-maven-plugin_${version.scala}</artifactId>
	<version>${version.scalor-maven-plugin}</version>
	<configuration>
		<defineBridge>
			<dependency>
				<groupId>org.scala-sbt</groupId>
				<artifactId>compiler-bridge_${version.scala}</artifactId>
				<version>${version.scala.zinc}</version>
			</dependency>
		</defineBridge>
		<defineCompiler>
			<dependency>
				<groupId>org.scala-lang</groupId>
				<artifactId>scala-compiler</artifactId>
				<version>${version.scalalibs}</version>
			</dependency>
		</defineCompiler>
		<definePluginList>
			<dependency>
				<groupId>com.artima.supersafe</groupId>
				<artifactId>supersafe_${version.scalalibs}</artifactId>
				<version>${version.supersafe}</version>
			</dependency>
		</definePluginList>
		<zincCompileOptions>
			-target:jvm-1.8
			-encoding
			${project.build.sourceEncoding}
		</zincCompileOptions>
	</configuration>
	<executions>
		<execution>

			<!-- Subset of available. -->
			<goals>
				<!-- Add compilation sources. -->
				<goal>register-macro</goal>
				<goal>register-main</goal>
				<goal>register-test</goal>

				<!-- Compile sources. -->
				<goal>compile-macro</goal>
				<goal>compile-main</goal>
				<goal>compile-test</goal>

			</goals>

		</execution>
	</executions>
</plugin>
```

I'ts trying to load sources:
```
org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact com.artima.supersafe:supersafe_2.12.4:jar:sources:1.1.3 in milton-repo (http://dl.bintray.com/milton/Milton)
```
which are not available...

Any hints?

**Comment by Andrei-Pozolotin** [comment]
please run with more debug info
http://maven.apache.org/ref/3.1.0/maven-embedder/cli.html
```
mvn -e -X ...
```
and attach sanitized logs zips with full stack traces here

**Comment by andreak** [comment]
Are you able to run with Supersafe plugin?

**Comment by Andrei-Pozolotin** [comment]
1. ok, that was a bug 

2. please try development release v 1.1.2
https://bintray.com/random-maven/maven/scalor-maven-plugin_2.12

**Comment by andreak** [comment]
Nice, it works now.
Thanks.

**Comment by Andrei-Pozolotin** [comment]
1. released 1.1.2
https://repo1.maven.org/maven2/com/carrotgarden/maven/scalor-maven-plugin_2.12/

2. please test again and close this if happy

**Comment by andreak** [comment]
Works.
